app/controllers/sessions_controller.rb

```diff
@@ -6,7 +6,7 @@ def create
     if user&.authenticate params[:session][:password]
       log_in user
       params[:session][:remember_me] == "1" ? remember(user) : forget(user)
-      redirect_to user
+      redirect_back_or user
```

Comment:
cái này để làm gì vậy em

Reply:
Khi muốn đến trang thực hiện hành động như chỉnh sửa hồ sơ thì phải login trước. Khi đó sẽ được chuyển đến trang log in. Sau khi thực hiện log in thì sẽ quay về trang chỉnh sửa hồ sơ mà ta muốn thực hiện. redirect_back_to thực hiện điều đó